core/tests/webdriverio_utils/StoryEditorPage.js

```diff
@@ -448,6 +449,8 @@ var StoryEditorPage = function() {
   };
 
   this.changeNodeOutline = async function(richTextInstructions) {
+    await browser.execute('window.scrollTo(0,500);');
```

Comment:
Is this necessary? We don't do it in our other tests so why do we need it here?

Reply:
I remove this.

Reply:
This seems to be still in the code, though? (It shows up in "Files Changed".)